addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -2,6 +2,10 @@
 .o_kanban_view {
     @include media-breakpoint-down(md) {
         --ControlPanel-border-bottom: none;
+
+        &.o_field_x2many_kanban .o_kanban_ghost {
+            display: none;
+        }
```

Comment:
Hello @odoo/rd-framework-js 

Can we have your opinion about that? 🙂 

Thanks in advance! 

Have a nice day! 🙂

Reply:
Not sure that I get the rationale behind this rule... :thinking:

Reply:
@pparidans what do you think? 🙂

Reply:
I see your point (remove the gap of the "ghost" records).
But there is a visual bug with 0 or 1 record : the "Add" button and/or the first record doesn't take the whole width.
![59639686-master-all runbot172 odoo com_web_id=93 action=818 active_id=5 model=project task view_type=form menu_id=583 cids=1 debug=assets(iPhone SE) (2)](https://github.com/odoo/odoo/assets/33146/d1aa18da-75ac-40b0-8959-402f89adcf44)
![59639686-master-all runbot172 odoo com_web_id=93 action=818 active_id=5 model=project task view_type=form menu_id=583 cids=1 debug=assets(iPhone SE) (1)](https://github.com/odoo/odoo/assets/33146/6e47c21f-b275-401a-9c4f-5a52977231cd)
![59639686-master-all runbot172 odoo com_web_id=93 action=818 active_id=5 model=project task view_type=form menu_id=583 cids=1 debug=assets(iPhone SE)](https://github.com/odoo/odoo/assets/33146/c9873b36-c1e9-42a3-a818-306666ee152f)

Reply:
Yup my bad here's my fix
Actually the Ghost records don't take any space, it's the gap styling of the kanban rendere that's adding vertical space between all the ghost records.
If there is 1 or 0 records, i remove the kanban gap (the gap isn't needed there since there is only one visible record) but i keep the kanban ghost records so the Add button displays correctly.

And if there's more than 1 record we keep the initial Kanban-gap to create space between the records, but we need to not display the kanban ghost records or they'll take the vertical space

https://github.com/odoo/odoo/blob/b629cb176e73eec9e877a935801bf601894eab96/addons/web/static/src/views/kanban/kanban_controller.scss#L5-L22

Is this good for you @pparidans ? tell me if i can remove the comments